rover/source/docs/sailboat-hardware.rst

```diff
@@ -19,10 +17,62 @@ to the wind speed the more important this is. See :ref:`wind-vane`.
 .. image:: ../images/sailboat-windvane.png
     :target: ../_images/sailboat-windvane.png
 
-Autopilots
+Sailing vehicles also require a method of controlling the sails. There are three supported actuator types, all actuators on a single vehicle should be of the same type.
```

Comment:
That would be the sane thing to do, but I don't see a technical reason you couldn't mix different types of sails on the same boat. You could, conceivably, have a direct-rotation rigid sail with a conventional jib.

Reply:
The heel PID will not let both out evenly, might end up with quite a odd result. We could fix in code for the direct wing and main-sheet by applying the PID as a angle rather than a offset but the wing sail elevator works differently again.